coursedata/adventures/de.yaml

```
@@ -1304,12 +1397,14 @@ adventures:
                     ## Restaurant
                      In level 1 you can make your own virtual restaurant and take your guests' orders.
 
+                example_code: |
                     ## Example Hedy code
+
                       ```
                      print Welcome to Hedy's restaurant 🍟
                      ask What would you like to order?
-                     echo So you would like to order 
-                     print Thank you for your order!
+                     echo So you would like to order
```

Keep that space, so that the echoed ordered thing(s) won't be glued to the word "order"!
```suggestion
                     echo So you would like to order 
```

---

See above